README.md

```diff
@@ -99,22 +101,54 @@ then [docker-compose-no-pleuston.yml](docker-compose-no-pleuston.yml) will be us
 If you do:
 
 ```bash
-./start_ocean.sh --latest --local-parity-node
+./start_ocean.sh --latest --local-secret-store
 ```
 
-then [docker-compose-local-parity-node.yml](docker-compose-local-parity-node.yml) will be used. Read it to see what images it starts. Note that it _doesn't_ start Pleuston, and it _does_ start a Parity Secret Store.
+then [docker-compose-local-secret-store.yml](docker-compose-local-secret-store.yml) will be used. Note that it _doesn't_ start Pleuston, and it _does_ start a Parity Secret Store. The images that are started would be:
+
+- mongo:3.6
+- oceanprotocol/parity-ethereum:master (for parity client)
+- oceanprotocol/parity-ethereum:master (for secret store)
+- nginx:alpine (for secret store cluster)
+- oceanprotocol/keeper-contracts:latest
+- oceanprotocol/aquarius:latest
+- oceanprotocol/brizo:latest
```

Comment:
I can't find the file `docker-compose-local-secret-store.yml` so I'm not able to verify that this is what's in it.